src/main/kotlin/no/nav/faktureringskomponenten/controller/dto/FakturaserieResponseDto.kt

```diff
@@ -33,7 +35,7 @@ data class FakturaserieResponseDto(
 
     val status: FakturaserieStatus,
 
-    val intervall: FakturaserieIntervallDto,
+    val intervall: FakturaserieIntervall,
```

Comment:
Hva er grunnen(e) til av vi endrer tilbake til domeneobjekt på en ResponseDto?


Årsak til at vi bør bruke DTO (selv om de er enkle enums) er:

- **Bedre organisering:** Ved å skille dataene i forskjellige lag kan koden organiseres bedre og blir lettere å vedlikeholde.

- **Bedre sikkerhet:** Hvis sensitive data ikke trenger å overføres til frontend, kan man hindre at uvedkommende får tilgang til dem.

- **Bedre skalerbarhet:** Hvis man separerer DTO og domenedata, kan man lettere skalere applikasjonen ved å distribuere forskjellige deler av koden på forskjellige servere.

- **Bedre fleksibilitet:** Hvis man senere ønsker å endre domenemodellen eller legge til flere funksjoner i frontend, vil det være lettere å gjøre endringene.

- **Bedre testing:** Ved å skille DTO og domenedata, kan man teste disse delene av koden uavhengig av hverandre og få bedre kontroll på testingen.

- **Bedre gjenbruk:** Hvis man separerer DTO og domenedata, kan man lettere gjenbruke koden på tvers av forskjellige prosjekter eller deler av applikasjonen.

Dette gjelder også ved Request DTOs.

Reply:
https://nav-it.slack.com/archives/C01BSCJM127/p1678864276920619

Reply:
Løst på https://nav-it.slack.com/archives/C01BSCJM127/p1678864276920619